src/views/pages/ManagePeople/index.vue

```diff
@@ -57,7 +57,7 @@ export default Vue.extend({
   created: async function created() {
     // get people types
     const personCategories: any[] = [];
-    await axios.get('https://boresha.tech:19443/people/categories')
+    await axios.get('https://boresha.live:19443/people/categories')
```

Comment:
i think in the long run it would be best to have some thing that deals with making requests so we don't have hard coded request set ups everywhere

Reply:
Yea, this is a larger configuration problem. Not part of this PR IMO